demo/halomi/LICENSE

```diff
@@ -0,0 +1,399 @@
+Attribution-NonCommercial 4.0 International
```

Comment:
is that correct? I don't think we can have two different licenses in one repo. If we put it as part of stopes, can we leave it as MIT?

Reply:
1. This license was a condition of the SRT approval of HalOmi release. Please ask Marta for more details. 
2. We already have a precedent of using a different license for a part of codebase: https://github.com/facebookresearch/stopes/tree/main/stopes/eval/alti